usagov/vote-gov-drupal
# VOTE-2802 NVRF updates

## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-2802

## Description

- Update NVRF to v0.17.0
- Remove the language selector from the NVRF page routes
- Update the NVRF Spanish page route path: /llenador-de-formulario-por-correo-postal

## Deployment and testing

### Post-deploy steps

1. cd in vote_nvrf and run `npm i`
2. run `lando drush cr`

### QA/Testing instructions

1. visit http://vote-gov.lndo.site/register/washington/mail-in-form-filler and test the latest verison of the NVRF and confirm it successfully creates an English PDF.
2. Confirm the language selector is removed on this page
3. visit http://vote-gov.lndo.site/es/registrar/washington/llenador-de-formulario-por-correo-postal and confirm that the NVRF loads on that page and confirm it successfully creates a Spanish PDF.

## Checklist for the Developer




## Checklist for the Peer Reviewers
